### PR TITLE
fix: Imported messages are not displayed

### PR DESCRIPTION
**Pull request by matheusbsilva137**
## Proposed changes (including videos or screenshots)

 - Fix issue with imported messages not being displayed in the room.
 - Fix importer agent being added as a member of every imported room.

## Issue(s)

Introduced in #28687 and #28695

## Further comments

Imported messages were added along with a `u._id` field, which caused them not to be displayed (messages would have a `u._id` field  and ALSO the `u`object with the `_id` and `username` fields).

TC-732

### CODE REVIEW

File: apps/meteor/app/lib/server/functions/createRoom.ts [hunk 1]
```diff
@@ -24,6 +24,7 @@ export const createRoom = async <T extends RoomType>(
 	name: T extends 'd' ? undefined : string,
 	ownerUsername: string | undefined,
 	members: T extends 'd' ? IUser[] : string[] = [],
+	excludeSelf?: boolean,
```
Comment (totoi690) [inline apps/meteor/app/lib/server/functions/createRoom.ts:27]: Couldn't we here transform all the optional parameters into a object so we don't have to add this `excludeSelf` in all other files?

Comment (pierre-lehnen-rc) [reply apps/meteor/app/lib/server/functions/createRoom.ts:27]: At least move the new param to the existing options object - also add a comment to the type definition to clarify what this param does, as "excludeSelf" by itself doesn't say much.

File: apps/meteor/app/lib/server/functions/insertMessage.ts [hunk 2]
```diff
@@ -32,8 +32,6 @@ export const insertMessage = async function (
 		}
 		await Messages.insertOne({
```
Comment (pierre-lehnen-rc) [inline apps/meteor/app/lib/server/functions/insertMessage.ts:33]: this insertOne is supposed to only run when `existingMessage` is undefined - can you add an `else` block around it?
as it is right now it'll be duplicating messages if the import is executed multiple times.

**Review by pierre-lehnen-rc — CHANGES_REQUESTED** [review]

File: apps/meteor/app/lib/server/functions/insertMessage.ts (hunk 2 above)
Comment (matheusbsilva137) [reply apps/meteor/app/lib/server/functions/insertMessage.ts]: Thankss, fixed!

File: apps/meteor/app/lib/server/functions/createRoom.ts (hunk 1 above)
Comment (matheusbsilva137) [reply apps/meteor/app/lib/server/functions/createRoom.ts:27]: I'll work on it in a separate PR just so that we do not hold this one for the patch release

**Review by gabriellsh — APPROVED** [review]

**Review by pierre-lehnen-rc — APPROVED** [review]

**Pull request merged**